docs/user-guide/global-configurations/gitops.md

```diff
@@ -137,7 +137,13 @@ Fill the following mandatory fields:
 | **Azure DevOps Username*** | Provide the username of your Azure DevOps account |
 | **Azure DevOps Access Token*** | Provide your Azure DevOps access token. It is used as an alternate password to authenticate your Azure DevOps account. <br />If you do not have one, create a Azure DevOps access token [here](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page). |
 
-### Bitbucket Cloud
+### Bitbucket
+
+Here, you get 2 options:
+* [Bitbucket Cloud](#bitbucket-cloud) - Select this if you wish to store GitOps configuration in a web-based Git repository hosting service offered by Bitbucket.
```

Comment:
some linking to bitbucket products may be helpful maybe

Reply:
I agree. We do have 3 links in the doc (the ones that might be relevant to our readers). They are all part of the product page, so product-related info is also easily accessible to the readers from there:

- https://support.atlassian.com/bitbucket-cloud/docs/group-repositories-into-projects/
- https://confluence.atlassian.com/bitbucketserver/creating-projects-776639848.html
- https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/